pettingzoo/classic/rps/rps.py

```diff
@@ -63,12 +75,101 @@ def reinit(self):
         self.num_moves = 0
 
     def render(self, mode="human"):
+        def offset(i, size):
+            if i == 0:
+                return -(size * (3 / 2))
+            else:
+                return size * (1 / 2)
+
+        screen_width = 800
+        screen_height = 1000
+
+        if self.screen is None:
+            if mode == "human":
+                pygame.init()
+                self.screen = pygame.display.set_mode((screen_width, screen_height))
+            else:
+                self.screen = pygame.Surface((screen_width, screen_height))
+        if mode == "human":
+            pygame.event.get()
+
+        # Load and all of the necessary images
+        paper = get_image(os.path.join('img', 'Paper.png'))
+        rock = get_image(os.path.join('img', 'Rock.png'))
+        scissors = get_image(os.path.join('img', 'Scissors.png'))
+        spock = get_image(os.path.join('img', 'Spock.png'))
+        lizard = get_image(os.path.join('img', 'Lizard.png'))
+
+        # Scale images in history
+        paper = pygame.transform.scale(paper, (int(screen_height / 9), int(screen_height / 9 * (14 / 12))))
+        rock = pygame.transform.scale(rock, (int(screen_height / 9), int(screen_height / 9 * (10 / 13))))
+        scissors = pygame.transform.scale(scissors, (int(screen_height / 9), int(screen_height / 9 * (14 / 13))))
+        spock = pygame.transform.scale(spock, (int(screen_height / 9), int(screen_height / 9)))
+        lizard = pygame.transform.scale(lizard, (int(screen_height / 9 * (9 / 18)), int(screen_height / 9)))
+
+        # Set background color
+        bg = (220, 220, 220)
+        self.screen.fill(bg)
+
+        if mode == "human":
+            # Set font properties
+            black = (0, 0, 0)
+            font = pygame.font.Font('freesansbold.ttf', 32)
```

Comment:
The font should render in rgb_array mode as well. I belive you have to call `pygame.font.init()` to make fonts work.